README.md

```diff
@@ -144,11 +158,25 @@ The purpose of this workflow is to parallelize the process of indexing files in
 1. Recreates the folder structure of the S3 bucket in the Synapse project.
 1. Indexes the files in the S3 bucket into the Synapse project.
 
+#### Workflow Diagram
+
+```mermaid
+flowchart LR
+    A[S3 Files] --> B[GET_USER_ID]
+    B --> C[UPDATE_OWNER]
+    C --> D[REGISTER_BUCKET]
+    A --> E[LIST_OBJECTS]
+    E --> F[SYNAPSE_MIRROR]
+    F --> G[SYNAPSE_INDEX]
+    D --> G
+    G --> H[Output CSV]
+```
+
 ### Quickstart:SYNINDEX
 
 The examples below demonstrate how you would index files from an S3 bucket called `example-bucket` into Synapse.
 
-1. Prepare your S3 bucket by setting the output directory of your general-purpose workflow to a Nextflow Tower S3 bucket. Ideally, you want this S3 bucket to be persistent (not a `scratch` bucket) so that your files will remain accessible indefinitely.
+1. Prepare your S3 bucket by setting the output directory of your general-purpose workflow to a Nextflow Tower S3 bucket. Ideally, you want this S3 bucket to be persistent (not a `-scratch` bucket) so that your files will remain accessible indefinitely.
```

Comment:
> _(not a `-scratch` bucket)_

Technically your output files can remain persistent in `scratch` buckets so long as they're under `work/` or any directory without a timeframe (e.g. `10days`) right?

This has me thinking, should we add a section for reasons why you would use `scratch` vs a regular bucket?

Reply:
No, the default lifespan for files within the entire scratch bucket is 90 days. /work is 30 and then everything beyond that is determined by the folder like /10days

Reply:
So I know that the /work directory has a default of 30 days and then you can further set expiration using the timeframe folder name like you said. I _thought_ that the whole bucket had a default (something like 90 days) but now I am wondering if that was wrong so I'll go check it out.

Reply:
Ah yeah here it is https://github.com/Sage-Bionetworks-Workflows/nextflow-infra/blob/1f3a328ea428efd0cabef13e373d5a03c0c9c164/templates/tower-project.j2#L484-L487

So it defaults to the max of these values https://github.com/Sage-Bionetworks-Workflows/nextflow-infra/blob/1f3a328ea428efd0cabef13e373d5a03c0c9c164/templates/tower-project.j2#L473-L474

For the entire bucket. So in summary:
- whole bucket defaults to 90 days
- bucket/work defaults to 30 days
- further expiration config can be done with timeframe folders like bucket/work/10days

Reply:
Ahh I see, good to know, thanks for the links!!